# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/service/VRServiceManager.cpp
# Commit: 898ed02804fe
# Full Hash: 898ed02804fefc9786112f135b9621cc444e6860
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r e1b7abc99ae9 -r 898ed02804fe gfx/vr/service/VRServiceManager.cpp
--- a/gfx/vr/service/VRServiceManager.cpp	Wed May 22 12:44:04 2019 +0000
+++ b/gfx/vr/service/VRServiceManager.cpp	Wed May 22 22:27:37 2019 +0000
@@ -7,6 +7,7 @@
 #include "VRServiceManager.h"
 #include "VRGPUChild.h"
 #include "mozilla/gfx/GPUParent.h"
+#include "mozilla/StaticPrefs.h"
 
 namespace mozilla {
 namespace gfx {
@@ -45,7 +46,7 @@
     }
     VRGPUChild::Shutdown();
   }
-  if (gfxPrefs::VRProcessEnabled()) {
+  if (StaticPrefs::VRProcessEnabled()) {
     // Using PGPU channel to tell the main process
     // to shutdown VR process.
     gfx::GPUParent* gpu = GPUParent::GetSingleton();
@@ -55,7 +56,7 @@
 }
 
 void VRServiceManager::CreateService() {
-  if (!gfxPrefs::VRProcessEnabled()) {
+  if (!StaticPrefs::VRProcessEnabled()) {
     mVRService = VRService::Create();
   }
 }